recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -338,7 +339,6 @@ The next steps are:
  4) Check the state of your device in
     your IoT Device Suite account" $WT_HEIGHT $WT_WIDTH
 		fi
-		do_restart_awsclient
```

Comment:
Could you explain the reasoning behind always restarting the awsclient when onboarding?

Why is the additional option to restart the awsclient needed? Can't you just use the `systemctl` command?

Can the notice about manually restarting the awsclient be removed after this change?